blueprint/core/blueprint_TimeoutFunctionManager.h

```diff
@@ -0,0 +1,128 @@
+#pragma once
```

Comment:
Let's keep this struct inside the EcmascriptEngine.cpp file since, truly, it is an implementation detail of the Duktape implementation of the EcmascriptEngine.

I think we can slim this down a bit so it doesn't feel like such a big blob of code for that file